docs/internal/contributing/README.md

```diff
@@ -56,12 +56,57 @@ make docker-image/pyroscope/build
 This target uses the `go/bin` target to first build binaries to include in the image.
 Make sure to pass the correct `GOOS` and `GOARCH` env variables.
 
-For example for `linux/amd64`:
-
+#### amd64 builds
 ```
 GOOS=linux GOARCH=amd64 make docker-image/pyroscope/build
```

Comment:
This only works because it is the default anyhow.

Either 

```suggestion
make docker-image/pyroscope/build
```

or 

```suggestion
make GOOS=linux GOARCH=amd64 docker-image/pyroscope/build
```